src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -417,7 +420,7 @@ protected function getFormActions()
                     throw new LogicException(get_class($this->record) . ' must implement ' . DataObjectInterface::class);
                 }
 
-                $noChangesClasses = 'btn-outline-primary font-icon-tick';
+                $noChangesClasses = $this->stagesDifferRecursive() ? 'btn-primary font-icon-save' : 'btn-outline-primary font-icon-tick';
```

Comment:
This check is needed to highlight “Save” if child versioned objects have changes